src/DependencyInjection/Admin/AutoConfiguredAdminInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\DependencyInjection\Admin;
+
+interface AutoConfiguredAdminInterface
+{
+    /**
+     * Return an array wuth all parameters required to configure admin:
```

Comment:
Not a fan of (untyped) arrays. Can we replace this with annotations or PHP8 attributes?

Reply:
Yes indeed, i'll update PR in that way